app/views/pages/crayons.html.erb

```diff
@@ -277,22 +277,54 @@
   </div>
   <div>
     <input type="text" class="crayons-textfield" placeholder="This is placeholder..."><br><br>
-    <input type="text" class="crayons-textfield" placeholder="This is placeholder..." value="Disabled field" disabled>
+    <input type="text" class="crayons-textfield" placeholder="This is placeholder..." value="Disabled field" disabled><br><br>
+    <input type="text" class="crayons-textfield crayons-textfield--ghost" placeholder="This is placeholder for ghost textfield...">
   </div>
 
   <pre><%= '
   <input type="text" class="crayons-textfield" placeholder="This is placeholder...">
   <input type="text" class="crayons-textfield" placeholder="This is placeholder..." value="Disabled field" disabled>
+  <input type="text" class="crayons-textfield crayons-textfield--ghost" placeholder="This is placeholder for ghost textfield...">
   '.rstrip %></pre>
 
   <div>
     <textarea class="crayons-textfield" placeholder="This is palceholder..."></textarea><br><br>
-    <textarea class="crayons-textfield" placeholder="This is palceholder..." disabled>Disabled textarea</textarea>
+    <textarea class="crayons-textfield" placeholder="This is palceholder..." disabled>Disabled textarea</textarea><br><br>
+    <textarea class="crayons-textfield crayons-textfield--ghost" placeholder="This is palceholder for ghost textarea..."></textarea>
   </div>
 
   <pre><%= '
   <textarea class="crayons-textfield" placeholder="This is palceholder..."></textarea>
   <textarea class="crayons-textfield" placeholder="This is palceholder..." disabled>Disabled textarea</textarea>
```

Comment:
LOL copy and paste typos in this file on the placeholder key ```placeholder="This is palceholder...">```

Reply:
:)